vision/getting-started/image_editing.ipynb

```diff
@@ -531,53 +589,14 @@
     "    prompt=PROMPT,\n",
     ")\n",
     "\n",
-    "local_file_path = \"tmp.png\"\n",
-    "response.images[0].save(local_file_path)\n",
-    "image_pil = Image.open(local_file_path)\n",
+    "response.images[0].save(IMAGE_TMP_FILE_PATH)\n",
```

Comment:
Unless you really need to save the image file, you should be able to access the Pil image from Imagen using `response.images[0]._pil_image`

https://github.com/googleapis/python-aiplatform/blob/bed3dec51f7e8991e6786d62ec721adf360555c4/vertexai/vision_models/_vision_models.py#L126

Reply:
We do need to save the image file. This is because the Gradio UI cell saves images to a tmp file so that the rest of the Colab cells can access it. Therefore, I've updated relevant cells to read from the temporary image file.

Reply:
Ok, that's fine